src/button-text-card.ts

```diff
@@ -53,7 +54,9 @@ export class BoilerplateCard extends LitElement {
       name: 'Button Text Card',
       title: '',
       subtitle: '',
+      height: '85px',
```

Comment:
For `icon_size` we don't require users to add `px`.
I think we shouldn't do it here either to stay uniform. What do you think?